core/build.gradle

```diff
@@ -19,24 +19,22 @@
 // THE SOFTWARE.
 
 description = 'Interfaces and utilities to report metrics to M3'
+apply from: 'jmhFixtures.gradle'
 
 sourceSets {
     jmh {
         java.srcDirs = ['src/jmh/java']
+//      explicitly disable excludes so that JMH tests from core module can be run
+        java.setExcludes([])
         resources.srcDirs = ['src/jmh/resources']
         compileClasspath += sourceSets.main.runtimeClasspath
         compileClasspath += sourceSets.test.runtimeClasspath
     }
 }
 
-dependencies {
-    jmhImplementation 'org.openjdk.jmh:jmh-core:1.23'
-    jmhImplementation 'org.openjdk.jmh:jmh-generator-annprocess:1.23'
-}
-
-task jmh(type: JavaExec, dependsOn: jmhClasses) {
+task runJmhTests(type: JavaExec, dependsOn: jmhClasses) {
```

Comment:
Nit: add README entry on running? (I'm assuming this is pretty standard but always helpful to have explicit things people can just copy paste).